--- gl2ps-1.4.2/CMakeLists.txt.orig	2022-09-11 15:02:34.057939204 +0200
+++ gl2ps-1.4.2/CMakeLists.txt	2022-09-11 15:04:19.237369398 +0200
@@ -143,7 +143,7 @@ if(OPENGL_FOUND)
   set_target_properties(lib PROPERTIES OUTPUT_NAME gl2ps)
 
   add_library(shared SHARED gl2ps.c gl2ps.h)
-  target_link_libraries(shared ${EXTERNAL_LIBRARIES})
+  target_link_libraries(shared ${EXTERNAL_LIBRARIES} m)
   set_target_properties(shared PROPERTIES OUTPUT_NAME gl2ps
     VERSION ${GL2PS_MAJOR_VERSION}.${GL2PS_MINOR_VERSION}.${GL2PS_PATCH_VERSION}
     SOVERSION ${GL2PS_MAJOR_VERSION})
@@ -171,9 +171,9 @@ install(FILES ${CMAKE_SOURCE_DIR}/gl2psT
 
 if(GLUT_FOUND)
   add_executable(gl2psTest WIN32 gl2psTest.c)
-  target_link_libraries(gl2psTest lib ${EXTERNAL_LIBRARIES})
+  target_link_libraries(gl2psTest shared ${EXTERNAL_LIBRARIES})
   add_executable(gl2psTestSimple WIN32 gl2psTestSimple.c)
-  target_link_libraries(gl2psTestSimple lib ${EXTERNAL_LIBRARIES})
+  target_link_libraries(gl2psTestSimple shared ${EXTERNAL_LIBRARIES})
 endif()
 
 find_package(LATEX)
